firstvoices/backend/models/import_jobs.py

```diff
@@ -116,3 +116,10 @@ class Meta:
     validation_report = models.OneToOneField(
         ImportJobReport, null=True, on_delete=models.SET_NULL
     )
+
+    failed_rows_csv = models.ForeignKey(
+        File,
+        null=True,
+        on_delete=models.SET_NULL,
+        related_name="import_job_failed_rows_csv_set",
+    )
```

Comment:
Could this be one-to-one instead of foreign key? It makes some things easier

Reply:
Kept as ForeignKey but updated the data field above in the same model to also be a ForeignKey as discussed.